facades/PC/server.build.gradle

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is still in Groovy because the embedded Groovy has a JsonBuilder.
+import groovy.json.JsonBuilder
+
+
+// By delaying this task to doLast (the << bit) we don't get the headless server dir set up unless actually wanting it
```

Comment:
Is this comment still up to date for the code? There is no notion of `<<` or `doLast` in the code (neither before nor after the refactoring)...

Reply:
I agree with your reading here.

I don't know whether that means it's just an obsolete comment that should be removed, or if there was a regression and the code lost some desired functionality.

Reply:
At this point we don't know (anymore), so let's remove this.
```suggestion
```